.pre-commit-config.yaml

```diff
@@ -1,11 +1,16 @@
-exclude: 'tests/test_data/'
+exclude: '.testhtml'
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.4.0
     hooks:
     -   id: check-yaml
     -   id: end-of-file-fixer
     -   id: trailing-whitespace
+    -   id: pretty-format-json
+        args:
+            - "--autofix"
+            - "--no-sort-keys"
+            - "--no-ensure-ascii"
```

Comment:
+100 on disabling the ASCII requirement.  So much nicer to read the contents of a bunch of the JSON test data as Unicode characters.